Pipfile

```diff
@@ -4,64 +4,63 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-aiokafka = "==0.5.2"
-django-environ = ">=0.4.0"
-djangorestframework = ">=3.8.0"
-gunicorn = ">=19.7.0"
-boto3 = "==1.10.2"
-whitenoise = ">=3.3.0"
+aiokafka = ">=0.5"
+django-environ = ">=0.4"
+djangorestframework = ">=3.11"
+gunicorn = ">=20.0"
+boto3 = ">=1.10"
+whitenoise = ">=5.0"
 Django = "~=2.2"
-psutil = "==5.6.3"
-drf-nested-routers = ">=0.90.0"
-django-filter = ">=1.1.0"
-requests = ">=2.20.0"
-django-tenant-schemas = "==1.10.0"
-django-cors-headers = "==3.1.1"
-querystring-parser = ">=1.2.3"
-djangorestframework-csv = "==2.1.0"
-pint = "==0.9"
-pytz = "==2019.3"
-django-prometheus = "==1.1.0"
-prometheus-client = "==0.7.1"
-django-redis = "==4.10.0"
-flake8 = "==3.7.9"
-watchtower = "==0.7.3"
-celery = "~=4.3.0"
-celery-prometheus-exporter = "==1.7.0"
-python-dateutil = "*"
-bs4 = "*"
-azure-mgmt-costmanagement = "==0.1.0"
-azure-mgmt-resource = "*"
-azure-mgmt-storage = "*"
-azure-storage-blob = "==2.1.0"
-azure-storage = "==0.36.0"
+psutil = ">=5.6"
+drf-nested-routers = ">=0.91"
+django-filter = ">=2.2"
+requests = ">=2.20"
+django-tenant-schemas = ">=1.10"
+django-cors-headers = ">=3.1"
+querystring-parser = ">=1.2"
+djangorestframework-csv = ">=2.1"
+pint = ">=0.9"
+pytz = ">=2019.3"
+django-prometheus = ">=1.1"
+prometheus-client = ">=0.7"
+django-redis = ">=4.10"
+flake8 = ">=3.7"
+watchtower = ">=0.7"
+celery = "~=4.4"
+celery-prometheus-exporter = ">=1.7"
+python-dateutil = ">=2.8"
+bs4 = ">=0.0.1"
+azure-mgmt-costmanagement = ">=0.1.0"
+azure-mgmt-resource = ">=8.0"
+azure-mgmt-storage = ">=7.1"
+azure-storage-blob = ">=12.1"
 google-cloud-storage = "~=1.19.1"
-jinjasql = "*"
-ciso8601 = "==2.1.2"
-pandas = "~=0.25.1"
-sentry-sdk = "~=0.13.2"
-ujson = "*"
+jinjasql = ">=0.1"
+ciso8601 = ">=2.1"
+pandas = ">=0.25"
+sentry-sdk = ">=0.13"
+ujson = ">=1.35"
 
 [dev-packages]
-astroid = "==2.3.2"
-pylint = "==2.4.3"
-sphinx-rtd-theme = ">=0.3.0"
-sphinx = ">=1.7.0"
-tox = ">=3.0.0"
+astroid = ">=2.3"
+pylint = ">=2.4"
+sphinx-rtd-theme = ">=0.3"
+sphinx = ">=1.7"
+tox = ">=3.0"
 coverage = "<5.0"
-codecov = ">=2.0.0"
-requests-mock = "==1.7.0"
-faker = ">=0.8.14"
-pylint-django = "==2.0.11"
-pydocstyle = "==4.0.1"
-flake8-docstrings = "*"
+codecov = ">=2.0"
+requests-mock = ">=1.7"
+faker = ">=0.8"
+pylint-django = ">=2.0"
+pydocstyle = ">=4.0"
+flake8-docstrings = ">=1.5"
 PyYaml = ">=5.3"
-tblib = "*"
-coveralls = "*"
-python-coveralls = "*"
-model-bakery = "*"
+tblib = ">=1.6"
+coveralls = ">=1.10"
+python-coveralls = ">=2.9"
+model-bakery = ">=1.1"
 koku-nise = "~=1.0"
-pre-commit = "*"
+pre-commit = ">=2.0"
 
 [requires]
-python_version = "3.6"
+python_version = ">=3.6"
```

Comment:
So, I get an error with this on mac:

```
cmyers-mac:koku cmyers$ pipenv --rm
Removing virtualenv (/Users/cmyers/.local/share/virtualenvs/koku-8P34cb70)…
cmyers-mac:koku cmyers$ pipenv install --dev --sequential
Traceback (most recent call last):
  File "/usr/local/Cellar/pipenv/2018.11.26_2/libexec/bin/pipenv", line 11, in <module>
    load_entry_point('pipenv==2018.11.26', 'console_scripts', 'pipenv')()
  File "/usr/local/Cellar/pipenv/2018.11.26_2/libexec/lib/python3.7/site-packages/pipenv/vendor/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/Cellar/pipenv/2018.11.26_2/libexec/lib/python3.7/site-packages/pipenv/vendor/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/Cellar/pipenv/2018.11.26_2/libexec/lib/python3.7/site-packages/pipenv/vendor/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/Cellar/pipenv/2018.11.26_2/libexec/lib/python3.7/site-packages/pipenv/vendor/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/Cellar/pipenv/2018.11.26_2/libexec/lib/python3.7/site-packages/pipenv/vendor/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/Cellar/pipenv/2018.11.26_2/libexec/lib/python3.7/site-packages/pipenv/vendor/click/decorators.py", line 64, in new_func
    return ctx.invoke(f, obj, *args, **kwargs)
  File "/usr/local/Cellar/pipenv/2018.11.26_2/libexec/lib/python3.7/site-packages/pipenv/vendor/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/Cellar/pipenv/2018.11.26_2/libexec/lib/python3.7/site-packages/pipenv/vendor/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/Cellar/pipenv/2018.11.26_2/libexec/lib/python3.7/site-packages/pipenv/cli/command.py", line 254, in install
    editable_packages=state.installstate.editables,
  File "/usr/local/Cellar/pipenv/2018.11.26_2/libexec/lib/python3.7/site-packages/pipenv/core.py", line 1741, in do_install
    pypi_mirror=pypi_mirror,
  File "/usr/local/Cellar/pipenv/2018.11.26_2/libexec/lib/python3.7/site-packages/pipenv/core.py", line 574, in ensure_project
    pypi_mirror=pypi_mirror,
  File "/usr/local/Cellar/pipenv/2018.11.26_2/libexec/lib/python3.7/site-packages/pipenv/core.py", line 494, in ensure_virtualenv
    python = ensure_python(three=three, python=python)
  File "/usr/local/Cellar/pipenv/2018.11.26_2/libexec/lib/python3.7/site-packages/pipenv/core.py", line 397, in ensure_python
    path_to_python = find_a_system_python(python)
  File "/usr/local/Cellar/pipenv/2018.11.26_2/libexec/lib/python3.7/site-packages/pipenv/core.py", line 360, in find_a_system_python
    python_entry = finder.find_python_version(line)
  File "/usr/local/Cellar/pipenv/2018.11.26_2/libexec/lib/python3.7/site-packages/pipenv/vendor/pythonfinder/pythonfinder.py", line 95, in find_python_version
    version_dict = PythonVersion.parse(major)
  File "/usr/local/Cellar/pipenv/2018.11.26_2/libexec/lib/python3.7/site-packages/pipenv/vendor/pythonfinder/models/python.py", line 359, in parse
    version_dict = parse_python_version(str(version))
  File "/usr/local/Cellar/pipenv/2018.11.26_2/libexec/lib/python3.7/site-packages/pipenv/vendor/pythonfinder/utils.py", line 86, in parse_python_version
    raise InvalidPythonVersion("%s is not a python version" % version_str)
pipenv.vendor.pythonfinder.exceptions.InvalidPythonVersion: >=3.6 is not a python version
```

Reply:
That's fine. It causes me errors on Fedora, because Fedora is on Python 3.7. But I don't mind working around it for a while longer.

Reply:
Update: https://github.com/pypa/pipenv/issues/1050
I think this is just a common pipenv thing, seems like devs want the ability to do it, but its a no go from the pipenv developers.

Reply:
@blentz, totally my default python version for mac is 3.7, but the pipenv should handle the python versioning mess for you. Are you using pipenv on fedora?